src/preact/base-element.js

```diff
@@ -227,6 +227,14 @@ PreactBaseElement['className'] = '';
  */
 PreactBaseElement['passthrough'] = false;
 
+/**
+ * Enabling attachShadow creates a shadow root without necessarily appending 'children'.
+ * This leaves the option for the component to define its 'children' prop itself,
+ * which may be of a non-prescriptive structure.
+ * @protected {boolean}
+ */
+PreactBaseElement['attachShadow'] = false;
```

Comment:
I don't think we need shadow in this case. I'd call this mode "disconnected". The safest way to approach it is to render into a `template.content`.

Reply:
Not sure I understand the suggestion. Do you mean not to render into the `PreactBaseElement.container_`?

Reply:
Yes. That's exactly the idea: to keep that DOM disconnected. It has no real use being connected - we just need the structure and components working on this structure.

Reply:
Correct me if I'm wrong but isn't its real use to field event listeners?